android-pdf-viewer/src/main/java/com/github/barteksc/pdfviewer/annotation/core/pdf/PdfUtil.kt

```diff
@@ -242,11 +243,13 @@ object PdfUtil {
                 page.render(bitmap, null, null, PdfRenderer.Page.RENDER_MODE_FOR_DISPLAY)
 
                 val pdfName = extractFileNameFromPath(pdfPath)
-                pngFile = saveBitmapAsPng(
-                    bitmap,
-                    outputDirectory,
-                    "PdfToImage-$pdfName-page-${pageNum + 1}.png"
-                )
+                pngFile = runBlocking {
+                    saveBitmapAsPng(
+                        bitmap,
+                        outputDirectory,
+                        "PdfToImage-$pdfName-page-${pageNum + 1}.png"
+                    )
+                }
```

Comment:
You are already in a suspend method you dont have to use `runBlocking` here - Gonna remove it while merging